src/vs/platform/issue/node/issueIpc.ts

```diff
@@ -29,7 +29,7 @@ export class IssueChannel implements IIssueChannel {
 			case 'openProcessExplorer':
 				return this.service.openProcessExplorer(arg);
 		}
-		return undefined;
+		return TPromise.as(null);
```

Comment:
We also have an engineering effort to remove `TPromise` (https://github.com/Microsoft/vscode/issues/53526), so let's use `Promise.resolve(null)` here instead